pkg/utils/utils.go

```diff
@@ -32,7 +32,7 @@ const (
 // COMPONENT_IMAGES are thee keys are based on the "container name" + "-{image,version}"
 var COMPONENT_IMAGES = map[string]string{
 	"curator":             "CURATOR_IMAGE",
```

Comment:
this was deleted in earlier commit in this PR, but added back in the subsequent commit